crypto/ocsp/ocsp_test.cc

```diff
@@ -822,6 +825,32 @@ TEST(OCSPRequestTest, AddCert) {
   certId = nullptr;
 }
 
+// Test that |OCSP_set_max_response_length| correctly sets the maximum response
+// length.
```

Comment:
Do we have a test that tests the expected behavior of max_resp_len? If the max response is really short or long that OCSP_REQ_CTX_nbio does the right thing?

Reply:
Sure, I added a test to set `max_resp_len` to a really short length, and test if `OCSP_REQ_CTX_nbio` fails as expected. Our existing tests already test that `OCSP_REQ_CTX_nbio` does the right thing, when max response has a sufficient length.